server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -380,6 +380,7 @@ public Phase next() {
     public static final int DEPENDENCIES_SEAM                           = 0x0A01;
     public static final int DEPENDENCIES_WS                             = 0x0C00;
     public static final int DEPENDENCIES_SECURITY                       = 0x0C50;
+    public static final int DEPENDENCIES_ELYTRON                        = 0x0C51;
```

Comment:
@dmlloyd Should there be a bigger gap here? I know you had some opinions on this in the past. On the other hand, this is pretty identical to security, so perhaps no gap is ok in this case.